tools/nemo_forced_aligner/utils/make_ass_files.py

```diff
@@ -183,17 +189,30 @@ def make_word_level_ass_file(
         "BorderStyle": "1",
         "Outline": "1",
         "Shadow": "0",
-        "Alignment": "2",
+        "Alignment": None,  # will specify below
         "MarginL": str(MARGINL),
         "MarginR": str(MARGINR),
-        "MarginV": str(ass_file_config.marginv),
+        "MarginV": str(MARGINV),
```

Comment:
Hardcoding?

Reply:
Yes, it's hardcoding from a variable I set at the top of the file. There are quite a lot of parameters such as this one that are currently hardcoded to sensible values. If there's demand, we can make it possible for the user to set them. But no need to do this right now.

Reply:
Sounds reasonable.